packages/@react-spectrum/picker/docs/Picker.mdx

```diff
@@ -0,0 +1,349 @@
+        items={options}
+        selectedKey={animal}
+        onSelectionChange={selected => setAnimal(selected)}>
+        {item => <Item uniqueKey={item.name}>{item.name}</Item>}
+      </Picker>
+    </>
+  );
+}
+```
+## Sections
+Picker supports sections in order to group options. Sections can be used by wrapping groups of Items in a `Section` component. Each `Section` takes a `title` and `uniqueKey` prop.
+
+### Static Items
+```tsx example
+<Picker label="Pick an animal">
+  <Section uniqueKey="Animals" title="Animals">
+    <Item uniqueKey="Aardvark">Aardvark</Item>
+    <Item uniqueKey="Kangaroo">Kangaroo</Item>
+    <Item uniqueKey="Snake">Snake</Item>
+  </Section>
+  <Section uniqueKey="People" title="People">
+    <Item uniqueKey="Danni">Danni</Item>
+    <Item uniqueKey="Devon">Devon</Item>
+    <Item uniqueKey="Ross">Ross</Item>
+  </Section>
+</Picker>
+```
+### Dynamic Items
+Sections used with dynamic items are populated from a hierarchical data structure. Similarly to the props on `Picker`, `Section` takes an array of data using the `items` prop.
+
+```tsx example
+function Example() {
+  let options = [
+    {name: 'Australian', children: [
+      {id: 2, name: 'Koala'},
+      {id: 3, name: 'Kangaroo'},
+      {id: 4, name: 'Platypus'}
+    ]},
+    {name: 'American', children: [
+      {id: 6, name: 'Bald Eagle'},
+      {id: 7, name: 'Bison'},
+      {id: 8, name: 'Skunk'}
+    ]}
+  ];
+
+  return (
+    <>
+      <Picker label="Pick an animal" items={options} onSelectionChange={selected => alert(selected)}>
+        {item => (
+          <Section uniqueKey={item.name} items={item.children} title={item.name}>
+            {item => <Item>{item.name}</Item>}
+          </Section>
+        )}
+      </Picker>
+    </>
+  );
+}
+```
+
+## Events
+Picker supports selection via mouse, keyboard, and touch. You can handle all of these via the `onSelectionChange`
+prop. Picker will pass the selected `uniqueKey` to the `onSelectionChange` handler.
+
+The following example uses an `onSelectionChange` handler to update the selection stored in React state.
+
+```tsx example
+function StaticExample() {
+  let [frequency, setFrequency] = React.useState();
+
+  return (
+    <>
+      <Picker label="Choose frequency" onSelectionChange={selected => setFrequency(selected)}>
+        <Item uniqueKey="Rarely">Rarely</Item>
+        <Item uniqueKey="Sometimes">Sometimes</Item>
+        <Item uniqueKey="Always">Always</Item>
+      </Picker>
+      <p>You selected {frequency}</p>
+    </>
+  );
+}
+```
+
+When using Picker with dynamic items, selection works much the same way using `uniqueKey`. However, if your data already has an `id` key (as is common with many data sets), Picker can use that id without needing to specify a `uniqueKey` prop. The below example shows Picker using the id of each item from the `items` array as the selected value without the need for `uniqueKey`. Note that `uniqueKey` will always take precedence if set.
```

Comment:
Maybe say "`id` property" instead of "key"? I feel like key has an overloaded meaning